src/main/java/org/broadinstitute/hellbender/engine/dataflow/transforms/composite/AddContextDataToReadOptimizedFn.java

```diff
@@ -0,0 +1,280 @@
+package org.broadinstitute.hellbender.engine.dataflow.transforms.composite;
```

Comment:
That wasn't an accident, I just wasn't sure where you wanted it. Moving to `engine.spark`.